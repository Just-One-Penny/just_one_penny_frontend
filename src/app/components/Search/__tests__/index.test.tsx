import React from 'react';
import { render } from '@testing-library/react';

import { Search } from '..';

describe('<Search  />', () => {
  it('should match snapshot', () => {
    const loadingIndicator = render(<Search />);
    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });
});
