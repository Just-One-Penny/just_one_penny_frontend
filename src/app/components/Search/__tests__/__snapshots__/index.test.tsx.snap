// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Search  /> should match snapshot 1`] = `
.c0 {
  background-color: #c0c2c4;
  height: 100px;
}

.c1 {
  width: 500px;
  height: 44px;
  background: #ffffff 0% 0% no-repeat padding-box;
  border: 1px solid #ffffff;
  border-radius: 50px;
  line-height: 1.5;
  -webkit-transition: border-color 0.15s ease-in-out,box-shadow 0.15s ease-in-out;
  transition: border-color 0.15s ease-in-out,box-shadow 0.15s ease-in-out;
  outline: none;
  opacity: 1;
  padding: 10px;
  margin-bottom: 10px;
  font-style: normal normal normal 16px/22px Avenir;
}

.c1::-webkit-input-placeholder {
  font-weight: 300;
}

.c1::-moz-placeholder {
  font-weight: 300;
}

.c1:-ms-input-placeholder {
  font-weight: 300;
}

.c1::placeholder {
  font-weight: 300;
}

.c1:hover {
  border: 1px solid #0a559e;
  opacity: 0.8;
}

.c1:active {
  opacity: 0.4;
  border: 0px;
}

.c2 {
  width: 255px;
  height: 44px;
  background: #ffffff 0% 0% no-repeat padding-box;
  border: 1px solid #ffffff;
  border-radius: 25px;
  opacity: 1;
  padding: 10px;
}

.c2:hover {
  border: 1px solid #0a559e;
}

.c2:focus {
  border: 1px solid #0a559e;
  border-radius: 25px;
}

.c3 {
  width: 170px;
  height: 44px;
  background: transparent linear-gradient(0deg,#11569b 0%,#54aecc 100%,#6ccddd 100%,#7ae0e8 100%) 0% 0% no-repeat padding-box;
  box-shadow: 0px 2px 6px #0000005a;
  border-radius: 50px;
  opacity: 1;
  margin-left: 80px;
  text-align: center;
  font: normal normal 900 16px/22px;
  color: #ffffff;
}

.c3:hover {
  opacity: 0.8;
}

.c3:active {
  opacity: 0.4;
}

.c3 .icon {
  margin-right: 0.25rem;
}

<div
  class="c0"
>
  <p>
    <input
      class="c1"
      id="charity"
      name="Charity"
      placeholder="Search Charity Name"
    />
  </p>
  <select
    class="c2"
    id="categories"
    name="Categories"
  >
    <option
      hidden=""
      value=""
    >
      Search By category
    </option>
    <option
      value="1"
    >
      Animal & Nature
    </option>
    <option
      value="2"
    >
      Arts & Education 
    </option>
    <option
      value="3"
    >
      Communities & Religion
    </option>
    <option
      value="4"
    >
      Human Outreach
    </option>
  </select>
  <button
    class="c3"
  >
    Find Charity
  </button>
</div>
`;
